msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -330,6 +332,7 @@ public static AcquireTokenNoFixedScopesCommandParameters createAcquireTokenNoFix
                 .authority(authority)
                 .authenticationScheme(authenticationScheme)
                 .forceRefresh(forceRefresh)
+                .scopes(scopes)
```

Comment:
Why do we still need `AcquireTokenNoFixedScopesCommandParameters` and `AcquireTokenNoFixedScopesCommand`? Now that we're adding scopes, aren't they the same as `SilentTokenCommandParameters`? Using the latter is how I envision the reusing of existing MSAL components working. Did you explore this avenue? If so, what are your findings?

Reply:
> In this set of PR, I will focus on multiple tokens and in the next PR I will remove the AcquireTokenNoFixedScopes* classes. 

What I mean is that to support multiple tokens, we should be using `SilentTokenCommand`. So if your goal with this PR is to explore how we can support multiple tokens, then I'd like to see the use of `SilentTokenCommand`. Unless you've found that using that existing command doesn't work - if so please outline your findings.

Reply:
@SaurabhMSFT this comment is still relevant. Why do we still need `AcquireTokenNoFixedScopesCommandParameters` and `AcquireTokenNoFixedScopesCommand`?

Reply:
I have removed *NoFixedScopes* classes